timed/employment/models.py

```diff
@@ -244,14 +252,7 @@ def calculate_worktime(self, start, end):
         6. The absences are all absences for this user within time frame
         7. The balance is the reported time plus the absences plus the
             overtime credit minus the expected worktime
-
```

Comment:
I'd have to look at what standard y'all wanna use but afaict something like this could help us keep helpful comments
```
Parameters:
-----------------
start: calculate worktime starting on given day.
end:   calculate worktime till given day

Return:
----------
tuple of 3 values reported, expected and delta in given
````

@winged do you know if we decided on a new preferred docs format?

Reply:
I honestly don't want to force anything. I think the rules in [PEP257](https://peps.python.org/pep-0257/) are just fine.

Only reason we had some weird syntax is so sphinx-doc (and for a while mypy as well I think) which we don't use anymore, could parse out the type hints. We don't need that anymore.

I'd suggest just using plain text description of the parameters as well as the return values. "Just make it clear what's going on" (and use wording according to PEP257 ofc)

Reply:
I've added back documentation about the return types:

https://github.com/adfinis/timed-backend/blob/4420bf0c089da08220a8d77c07425a4c7eedb2a2/timed/employment/models.py#L256

https://github.com/adfinis/timed-backend/blob/4420bf0c089da08220a8d77c07425a4c7eedb2a2/timed/employment/models.py#L378